src/test/scala/performanceanalysis/administrator/LogParserActorSpec.scala

```diff
@@ -0,0 +1,31 @@
+package performanceanalysis.administrator
+
+import akka.actor.ActorSystem
+import akka.testkit.TestActorRef
+import performanceanalysis.base.ActorSpecBase
+import performanceanalysis.server.Protocol.{Details, Metric, MetricRegistered, RequestDetails}
+
+class LogParserActorSpec(_system: ActorSystem) extends ActorSpecBase(_system) {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Field name '_system' does not match the regular expression '^[a-z][A-Za-z0-9]*$'](https://www.codacy.com/app/scala-academy/performance-analysis/file/2431820052/issues/source?bid=3220915&fileBranchId=3241725#l8)